### ISSUE TITLE
Does not render correctly after first SAML message recorded

### ISSUE THREAD

**Issue by thijskh** [magnussuther/saml-message-decoder-extension#3]
Hi, I'm trying out the extension. But after receiving just one SAML message, the window does not appear, only a very thin grey line between address bar and content area:
![image](https://user-images.githubusercontent.com/3808792/31386393-f77c9642-adc6-11e7-995f-da0a4b2a3d13.png)

Not sure if this is fixed in the most recent commit, I got the most recent version from Firefox Addons which is 2.0.0, but there seem to be newer versions in the repo.

If this is already fixed, could you upload 2.0.4 (?) to Mozilla?
Besides I was on the lookout for build instructions so I could try out master locally. Could you add that to the readme?

**Comment by DanielMalmgren** [comment]
Same problem here. Since FF 57.0 seems to have been rolled out today and SAML Tracer doesn't work anymore lots of people (like me) will be looking for an alternative, hope this gets fixed so this one is usable...

**Comment by Yoshyn** [comment]
When import the module directly with about:debugging (after cloning)

Here is the error message retrieve from the browser console :
```
SyntaxError: import declarations may only appear at top level of a module  listener.js:11
```

Here is the most relevant response on stackoverflow for this is : https://stackoverflow.com/questions/37624819/es2015-import-doesnt-work-even-at-top-level-in-firefox

Sadly even with `dom.moduleScripts.enabled` set to true it does not work.

Bit if i run `npm run build-dev` and import the module directly with about:debugging : it work !

@magnussuther : What have you publish under `https://addons.mozilla.org/fr/firefox/addon/saml-message-decoder-extension` ?

**Comment by magnussuther** [comment]
Sorry for the radio silence. I made some changes, and just released v2.1.0. I haven't tested it against the latest FF though. Drop me a new Issue if you find any problems.

**Comment by Yoshyn** [comment]
Work for me now on FF 57.0. 
Moreover, the scrolling management is clearly better ;)
